languages/python/exercises/concept/strings/.docs/instructions.md

```diff
@@ -0,0 +1,45 @@
+
+In this exercise you'll be processing log-lines.
+
+Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+
+There are three different log levels:
+
+- `INFO`
+- `WARNING`
+- `ERROR`
+
+You have three tasks, each of which will take a log line and ask you to do something with it.
+
+### 1. Extract a message from a log line
+
+Implement a function to return a log line's message:
+
+```python
+extract_message("[ERROR]: Invalid operation")
+// Returns: "Invalid operation"
+```
+
+The message should be trimmed of any whitespace.
+
+```python
+extract_message("[ERROR]: Invalid operation.\t\n")
+// Returns: "Invalid operation."
+
```

Comment:
```suggestion
```

```

Reply:
Doh, never mind.  Was going to suggest the backticks, and realized a moment later than we can't do that in this way.  Not sure how to do so at the moment either.